src/helics/application_api/FederateInfo.cpp

```diff
@@ -37,37 +38,64 @@ FederateInfo::FederateInfo(const std::string& args)
     loadInfoFromArgsIgnoreOutput(args);
 }
 
-static const std::map<std::string, int> propStringsTranslations{
+static const std::unordered_map<std::string, int> propStringsTranslations{
     {"period", helics_property_time_period},
+    {"timeperiod", helics_property_time_period},
+    {"timeperiod", helics_property_time_period},
     {"timedelta", helics_property_time_delta},
     {"time_delta", helics_property_time_delta},
     {"timeDelta", helics_property_time_delta},
     {"delta", helics_property_time_delta},
     {"offset", helics_property_time_offset},
+    {"timeoffset", helics_property_time_offset},
+    {"time_offset", helics_property_time_offset},
     {"rtlead", helics_property_time_rt_lead},
     {"rtlag", helics_property_time_rt_lag},
     {"rttolerance", helics_property_time_rt_tolerance},
+    {"timertlead", helics_property_time_rt_lead},
+    {"timertlag", helics_property_time_rt_lag},
+    {"timerttolerance", helics_property_time_rt_tolerance},
     {"rtLead", helics_property_time_rt_lead},
     {"rtLag", helics_property_time_rt_lag},
     {"rtTolerance", helics_property_time_rt_tolerance},
     {"rt_lead", helics_property_time_rt_lead},
     {"rt_lag", helics_property_time_rt_lag},
     {"rt_tolerance", helics_property_time_rt_tolerance},
+    {"time_rt_lead", helics_property_time_rt_lead},
+    {"time_rt_lag", helics_property_time_rt_lag},
+    {"time_rt_tolerance", helics_property_time_rt_tolerance},
     {"inputdelay", helics_property_time_input_delay},
     {"outputdelay", helics_property_time_output_delay},
     {"inputDelay", helics_property_time_input_delay},
     {"outputDelay", helics_property_time_output_delay},
     {"input_delay", helics_property_time_input_delay},
     {"output_delay", helics_property_time_output_delay},
     {"max_iterations", helics_property_int_max_iterations},
+    {"timeinputdelay", helics_property_time_input_delay},
+    {"timeoutputdelay", helics_property_time_output_delay},
+    {"time_input_delay", helics_property_time_input_delay},
+    {"time_output_delay", helics_property_time_output_delay},
     {"loglevel", helics_property_int_log_level},
     {"log_level", helics_property_int_log_level},
-    {"maxiterations", helics_property_int_max_iterations},
     {"logLevel", helics_property_int_log_level},
+    {"intloglevel", helics_property_int_log_level},
+    {"intlog_level", helics_property_int_log_level},
```

Comment:
Why are we adding `intlog_level` and not `int_log_level` when later we are adding `int_file_log_level`. 

I think we shouldn't have any `int{}` or `time{}` variant. Basically, I would like to only support users entering the attribute as we use it in the code but without the `helics_property_time_` or `helics_property_int_` prefix.

I think there should be just one and obvious way to use a property. For example, I think it is confusing for users whether `iterations` and `max_iterations` are the same thing or not

Reply:
The "int" and "time" was what you were testing in the issue this is a based on.  Also in helics3 @trevorhardy  removed a lot of the variants so when I port this over, many will be removed.  I can remove them happily but then it won't technically be resolving the issue?

Reply:
@kdheepak   Do you think we should have the "time" and "int" variants like suggested in #1642 ?